src/compiler/commandLineParser.ts

```diff
@@ -493,8 +493,9 @@ namespace ts {
       * @param basePath A root directory to resolve relative path entries in the config
       *    file to. e.g. outDir
       */
-    export function parseJsonConfigFileContent(json: any, host: ParseConfigHost, basePath: string, existingOptions: CompilerOptions = {}): ParsedCommandLine {

```

Comment:
this is going to be an API breaking change. can we add an optional argument for the filename?